### PR TITLE
Chore/dep patch

### PR DESCRIPTION
**Pull request by mfshao**
Jira Ticket: [FRC-35](https://ctds-planx.atlassian.net/browse/FRC-35)

deployed in https://mingfei.planx-pla.net/

Update dependencies to patch FedRAMP vulnerabilities
- [X] Upgrade tar to version 6.1.9, 5.0.10, 4.4.18 or higher.
- [x] Upgrade prismjs to version 1.21.0 or higher.
- [X] Upgrade trim-newlines to version 3.0.1, 4.0.1 or higher.
- [X] Upgrade object-path to version 0.11.8 or higher.
- [X] Upgrade zrender to version 5.2.1 or higher.
- [X] Upgrade ansi-regex to version 6.0.1, 5.0.1 or higher.
- [X] Upgrade css-what to version 5.0.1 or higher.
- [X] Upgrade nth-check to version 2.0.1 or higher.
- [X] Upgrade path-parse to version 1.0.7 or higher.
- [X] Upgrade tmpl to version 1.0.5 or higher.
- [X] Upgrade xss to version 1.0.10 or higher.

### Dependency updates
- Upgrade tar to version 6.1.9, 5.0.10, 4.4.18 or higher.
- Upgrade prismjs to version 1.21.0 or higher.
- Upgrade trim-newlines to version 3.0.1, 4.0.1 or higher.
- Upgrade object-path to version 0.11.8 or higher.
- Upgrade zrender to version 5.2.1 or higher.
- Upgrade ansi-regex to version 6.0.1, 5.0.1 or higher.
- Upgrade css-what to version 5.0.1 or higher.
- Upgrade nth-check to version 2.0.1 or higher.
- Upgrade path-parse to version 1.0.7 or higher.
- Upgrade tmpl to version 1.0.5 or higher.
- Upgrade xss to version 1.0.10 or higher.

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -255,7 +254,8 @@ module.exports = {
     },
     {
       test: /\.svg$/,
-      loaders: ['babel-loader', 'react-svg-loader'],
+      // loaders: ['babel-loader', 'react-svg-loader'], // to address the `css-what` vulnerability issue, after updating to webpack 5 and latest `react-svg-loader` we can switch back to this
```
Comment (paulineribeyre) [inline webpack.config.js:257]: we might need the same comment to explain [here](https://github.com/uc-cdis/data-portal/blob/f20b3d2c7fca4cf8e5b4b3adfb61cc14d1a26bbf/.storybook/webpack.config.js#L39)?

File: Dockerfile [hunk 2]
```diff
@@ -27,8 +27,7 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
     && rm -rf /var/lib/apt/lists/* \
     && ln -sf /dev/stdout /var/log/nginx/access.log \
     && ln -sf /dev/stderr /var/log/nginx/error.log \
-    && npm install -g npm@7 \
-    && npm config set maxsockets 5
```
Comment (paulineribeyre) [inline Dockerfile:31]: curious why we don't need this anymore?

**Review by paulineribeyre — COMMENTED** [review]
lgtm, just need to update gen3-ui-component and guppy

File: webpack.config.js (hunk 1 above)
Comment (mfshao) [reply webpack.config.js:257]: nice catch, added that

File: Dockerfile (hunk 2 above)
Comment (mfshao) [reply Dockerfile:31]: this issue has been addressed in a newer version of npm https://github.com/npm/cli/issues/2977#issuecomment-809704961

**Review by paulineribeyre — APPROVED** [review]

**Pull request merged**